cdc/scheduler/internal/tp/schedulepb/table_schedule_manual_test.go

```diff
@@ -14,77 +14,87 @@
 package schedulepb
 
 import (
-	fmt "fmt"
-	testing "testing"
+	"fmt"
+	"testing"
 
 	"github.com/gogo/protobuf/proto"
+	"github.com/pingcap/tiflow/cdc/model"
 )
 
-func benchmarkMessageCheckpoints(b *testing.B, bench func(b *testing.B, m *Message)) {
+func benchmarkMessageHeartbeatResponse(b *testing.B, bench func(b *testing.B, m *Message)) {
 	size := 16384
 	for total := 1; total <= size; total *= 2 {
 		msg := Message{
-			Checkpoints: map[int64]Checkpoint{},
+			MsgType: MsgHeartbeatResponse,
+			HeartbeatResponse: &HeartbeatResponse{
+				Tables:     make([]TableStatus, 0, total),
+				IsStopping: false,
+			},
 		}
+
 		for i := 0; i < total; i++ {
-			msg.Checkpoints[int64(10000+i)] = Checkpoint{
-				CheckpointTs: 433331421532337260,
-				ResolvedTs:   433331421532337261,
-			}
+			msg.HeartbeatResponse.Tables = append(msg.HeartbeatResponse.Tables,
+				TableStatus{
+					TableID: model.TableID(i),
```

Comment:
```suggestion
					TableID: model.TableID(10000+i),
```

Let's close to the real case. Table ID starts from a large number for the this case.